src/website/app/pages/index.js

```diff
@@ -66,13 +73,17 @@ const sections: Array<Section> = [
         component: Loadable({
           loader: () => import('./ComponentStatus')
         }),
+        description:
+          'Check back here anytime to see current component status information for Mineral UI! Check our GitHub for issues or to suggest a new feature!',
         path: '/component-status',
         title: 'Component Status'
       },
       {
         component: Loadable({
           loader: () => import('./Roadmap')
         }),
+        description:
+          'As Mineral UI continues to grow, we want to keep you up to date. Let’s plan the future together!',
```

Comment:
This doesn't inspire confidence re: predictability and stableness, which I feel like is a primary goal for this page. Maybe something like:

> Mineral UI is committed to stable and predictable releases. Learn more about our plans for the future.